docs/buildah-source-push.1.md

```diff
@@ -20,6 +20,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--digestfile** *Digestfile*
```

Comment:
(Not a new thing but why is Digestfile capitalized?  It's not a well known thing like Dockerfile...)

Reply:
Removed, I think it got copied from older man-page, will fix it up there as well.